web/api/v1/api_test.go

```diff
@@ -3298,8 +3223,9 @@ func BenchmarkRespond(b *testing.B) {
 	}
 	b.ResetTimer()
 	api := API{}
```

Comment:
Suggest `api.InstallCodec(JSONCodec{})` on next line.

Reply:
Isn't that already there?

Reply:
Well that's weird.  According to my memory it there wasn't when I commented, but it is now.

I hit the problem in this other copy of the code: https://github.com/grafana/mimir-prometheus/blob/5c270e23eab7d8ca7f5b199e6e5543547c340244/web/api/v1/api_test.go#L3294-L3296

Reply:
Weird 🤷 

Good catch on mimir-prometheus being out of sync, I've just double-checked and I don't believe there are any other cases where something is missing. I'll fix this in mimir-prometheus tomorrow.